.github/workflows/velox_backend.yml

```diff
@@ -68,12 +70,24 @@ jobs:
           path: |
             ./cpp/build/releases/
           key: cache-velox-build-centos-7-${{ hashFiles('./cache-key') }}
+      - name: Get Ccache
+        uses: actions/cache/restore@v3
+        with:
+          path: '${{ env.CCACHE_DIR }}'
+          key: ccache-centos7-release-default
       - name: Build Gluten native libraries
         if: ${{ steps.cache.outputs.cache-hit != 'true' }}
         run: |
           df -a
+          yum install ccache -y
```

Comment:
ccache not installed on centos7 
CC @PHILO-HE

Reply:
Yes, will make it installed in another pr. Thanks!